py3status/modules/online_status.py

```diff
@@ -21,13 +20,7 @@
 @author obb
 """
 
-import os
-import subprocess
-try:
-    # python3
-    from urllib.request import urlopen
-except:
-    from urllib2 import urlopen
+from urllib.parse import urlsplit
```

Comment:
> Travis say this is okay for python 2.7? 

the syntax is valid but no module will be found

```
try:
    # python 3
    from urllib.parse import urlsplit
except ImportError:
    # python 2
    from urparse import urlsplit
```
everyone is happy.

It is quite useful for development if you can switch between python2/3 for testing.  I just copied `/usr/bin/py3status` to `/usr/bin/py3status2` and changed the #! line to point to python2 so I can easily run both.